docs/docs/sdk/prediction-import/kitti.mdx

```diff
@@ -3,23 +3,22 @@ title: KITTI Bounding Boxes
 sidebar_position: 2
 ---
 
-:::caution
-
-This works for bounding boxes only.
-
-:::
+The KITTI dataset is a widely-used computer vision dataset for benchmarking and evaluating autonomous driving systems.
+It consists of a collection of high-resolution images, lidar data, and camera calibration parameters captured from a moving platform driving through urban environments.
+The dataset includes a variety of tasks such as object detection, tracking, 3D object detection, and semantic segmentation.
```

Comment:
What dataset? We only care about the format here I think. Otherwise, perhaps say 
```suggestion
The [original dataset](https://www.cvlibs.net/datasets/kitti/) includes a variety of tasks such as object detection, tracking, 3D object detection, and semantic segmentation.
```

Reply:
Just giving an introduction for those that are looking for ways to import their data so they know it's not a fit.

Reply:
I rewrote that section to actually refer to the label format instead of the dataset.